docs/build-with-0g/compute-network/sdk.md

```diff
@@ -218,69 +175,41 @@ const completion = await openai.chat.completions.create(
 );
 ```
 
-**Note**: After receiving the response, you must use `processResponse` as demonstrated in step 5.4 to settle the response fee. Failure to do so will result in subsequent requests being denied due to unpaid fees. If this happens, you can manually settle the fee using `settleFee` as shown in step 5.5. The amount owed will be specified in the error message.
-
-**Note**: Generated `headers` are valid for a single use only and cannot be reused.
+### Response Processing
 
-#### 5.4 Process Responses
+This function is used to verify the response and settle the fee. If it is a verifiable service, it will return whether the response is valid.
```

Comment:
Can we explain a little more what constitutes a "verifiable service"? cc @Ravenyjh

Reply:
Good suggestion, we can include this content: 

"When the 'verifiability' attribute of a service is not empty, the service can be verified on the client side. We currently support 'TeeML' (with 'OPML' and 'ZKML' anticipated in the future). 'TeeML' allows each response to be verified as executed in a Trusted Execution Environment using the 'processResponse' function, as detailed here: [processResponse](https://github.com/0glabs/0g-serving-user-broker/blob/f99b523d85daa3be4e2dabac2caa31c2feba4b89/src.ts/broker/response.ts#L78)."